Project.toml

```
@@ -12,4 +13,5 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
 Adapt = "2.0, 3.0"
+LLVM = "4.7"
```

`assume` was added in LLVM.jl 3.9.0

```suggestion
LLVM = "3.9"
```

---

Shouldn't it be `3.9, 4` ? https://pkgdocs.julialang.org/v1/compatibility/#Compatibility

---

Oh yes, of course, good catch :slightly_smiling_face: